packages/google_sign_in/google_sign_in_web/lib/google_sign_in_web.dart

```diff
@@ -128,8 +128,15 @@ class GoogleSignInPlugin extends GoogleSignInPlatform {
   @override
   Future<GoogleSignInUserData> signIn() async {
     await initialized;
-
-    return gapiUserToPluginUserData(await auth2.getAuthInstance().signIn());
+    try {
+      return gapiUserToPluginUserData(await auth2.getAuthInstance().signIn());
+    } on auth2.GoogleAuthSignInError catch (reason) {
+      throw PlatformException(
+        code: reason.error,
+        message: 'Exception raised from GoogleAuth.signIn()',
+        details: 'google_sign_in',
```

Comment:
`google_sign_in` as the details doesn't seem to be necessary. Maybe change the message to 
`google_sign_in: Exception raised from GoogleAuth.signIn()` 
And for details, we can put more details if we have (stack trace etc)

Reply:
The only guaranteed information we have from the JS SDK here is the ["error" property](https://developers.google.com/identity/sign-in/web/reference#googleauthsignin) that we're using as the code.

Maybe in `details` I can add the link to the JS SDK so people can understand what the error means? (the values are pretty self-explanatory, though)

Reply:
Ended up adding the link!